app/keyword/Card.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { useState, useEffect }  from 'react';
+import Image from 'next/image';
+
+interface CardProps {
+  card: {
+    subtitle: string;
+    title: string;
+    description: string;
+    image: string;
+  };
+}
+
+const Card: React.FC<CardProps> = ({ card }) => {
```

Comment:
There is an opinion that we should avoid using React.FC. I think it would be good to refer to it.
[link](https://github.com/facebook/create-react-app/pull/8177)